Makefile.am

```diff
@@ -40,15 +40,15 @@ test: check-recursive
 
 # Target to make autogenerated files.
 checkprograms :
-	(cd src ; $(MAKE) libsndfile.la checkprograms)
-	(cd tests ; $(MAKE) checkprograms)
+	$(MAKE) -C src libsndfile.la checkprograms
+	$(MAKE) -C tests checkprograms
```

Comment:
I am aware of this way of doing it, but why is this better? What happens if I accept this patch and later get a patch to change it back?